index.html

```diff
@@ -2905,6 +2915,166 @@ <h3>Evidence</h3>
+<a>verifiable presentation</a>, where the verification key of the cryptographic
+signature is bound to a confirmation method in the embedded <a>verifiable
+credential</a>.
+          </p>
+          <p>
+A <a>verifier</a> can validate that the <a>holder</a> controls,
+or has been designated the ability to use, a confirmation method
+by verifying the <a>proof</a> of the <a>verifiable
+presentation</a> using the information in the confirmation method. The
+confirmation method can include the verification key, or the type of the
+confirmation method can define that the verification key is to be inferred from
+other <a>properties</a> in the <a>verifiable credential</a>, such as the
+<code>credentialSubject.id</code>.
+          </p>
+        </div>
+
+        <dl>
+          <dt><dfn>confirmationMethod</dfn></dt>
+          <dd>
+          <dd>
+            <p>
+If present, the value of the <code>confirmationMethod</code> <a>property</a> is
+one or more confirmation methods. Each confirmation method is bound to one or more
+claims in the <a>verifiable credential</a>, and provides enough information for a
+<a>verifier</a> to confirm whether the <a>holder</a> generating a <a>verifiable
+presentation</a> can satisfy that confirmation method. It is required that the
+<a>issuer</a> verifies
+the <a>holder</a> can satisfy each <code>confirmationMethod</code> they include
+in the <a>claims</a> of the <a>verifiable credential</a>s they issue.
+            </p>
+            <p>
+Each confirmation method MUST specify its <code>type</code> (for example,
+<code>DIDAuthWithSubjectIdConfirmation</code>) and MAY specify an
+<code>id</code>. The precise <a>properties</a> and semantics of each
+confirmation method are determined by the specific
+<code>confirmationMethod</code> type definition.
+            </p>
+          </dd>
+        </dl>
+
+        <p>
+The type <code>DIDAuthWithSubjectIdConfirmation</code> defines that a
+<a>verifier</a> MAY validate the confirmation method by verifying the proof of
+the <a>verifiable presentation</a> with the verification material of one of the
+authentication verification relationships of the
+<code>credentialSubject.id</code> if the particular
+<code>credentialSubject.id</code> is a Decentralized Identifier (DID).
+        </p>
+
+       <pre class="example nohighlight"
+          title="Usage of the confirmationMethod property of type DIDAuthWithSubjectIdConfirmation">
+{
+  "@context": [
+    "https://www.w3.org/ns/credentials/v2",
+    "https://www.w3.org/ns/credentials/examples/v2"
+  ],
+  "id": "http://example.edu/credentials/3732",
+  "type": ["VerifiableCredential", "UniversityDegreeCredential"],
+  "issuer": "https://example.edu/issuers/14",
+  "validFrom": "2010-01-01T19:23:24Z",
+  "credentialSubject": {
+    "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+    <span class="highlight">"confirmationMethod": {
+      "type": "DIDAuthWithSubjectIdConfirmation"
```

Comment:
Please add an issue noting that we're still bikeshedding the `confirmationMethod` name and the `DIDAuthWithSubjectIdConfirmation` name.

Reply:
I guess this is resolved. I'll remove the references to `DIDAuthWithSubjectIdConfirmation`.